backend/tests/unit/test_graph_builder.py

```
@@ -0,0 +1,595 @@
+import pytest
+from pathlib import Path
+from typing import Dict, List, Any
```

Ruff (0.12.2)

`typing.Dict` imported but unused

Remove unused import

(F401)